liquibase-standard/src/main/java/liquibase/command/core/GenerateChangelogCommandStep.java

```diff
@@ -68,6 +76,12 @@ public class GenerateChangelogCommandStep extends AbstractCommandStep {
                 .description("Directory to write table data to").build();
         OVERWRITE_OUTPUT_FILE_ARG = builder.argument("overwriteOutputFile", Boolean.class)
                 .defaultValue(false).description("Flag to allow overwriting of output changelog file. Default: false").build();
+        RUNONCHANGE_TYPES_ARG = builder.argument("runOnChangeTypes", String.class)
+                .defaultValue("none").description("Sets runOnChange=\"true\" for changesets containing solely changes of these types (e. g. createView, createProcedure, ...).").build();
+        final String replaceIfExistsTypeNames = supportedReplaceIfExistsTypes().collect(Collectors.joining(", "));
+        REPLACEIFEXISTS_TYPES_ARG = builder.argument("replaceIfExistsTypes", String.class)
+                .defaultValue("none")
+                .description(String.format("Sets replaceIfExists=\"true\" for changes of these types (supported types: createView, createProcedure)", replaceIfExistsTypeNames)).build();
```

Comment:
I think we are missing the '%s' on the provided text here.

Reply:
You are right, I have just corrected this in 0e959aea1. Thank you for reporting it!